cupy/cuda/cudnn.pxd

```diff
@@ -133,7 +133,9 @@ cpdef destroyTensorDescriptor(size_t tensorDesc)
 cpdef addTensor_v2(
         size_t handle, int mode, size_t alpha, size_t biasDesc,
         size_t biasData, size_t beta, size_t srcDestDesc, size_t srcDestData)
-
+cpdef addTensor(
```

Comment:
We had prepared `addTensor_v3` in the master branch (it is actually identical to `addTensor` if cuDNN is newer then v2). So please merge or rebase the master branch and use this API.